### PR TITLE
#CP-21 feat: reset password integration

### PR DESCRIPTION
**Pull request by Abdulkeza**
## What does this PR do?
- Reset the password for the users
## Description of Task to be completed?

- [x] The user should be able to provide their email.
- [x] The driver or operator should be able to log in with the new password.
- [x] The driver or operator should be able to receive a reset password link in their email.
- [x] The driver or operator should be able to reset the password( i.e. provide a new password).
- [x] All Requests should be sent to the backend and return appropriate responses
- [x] The system should work with real data from the database

## How should this be manually tested?
- [Clone](https://github.com/atlp-rwanda/phantom-fe-codebandits.git)
- git co ft-reset-password-cp-21
- npm i
- npm t
 
## Any background context you want to provide?
(N/A)

## What are the relevant pivotal tracker stories?
[SUz0qwab](https://trello.com/c/SUz0qwab/21-reset-password)

### CODE REVIEW

File: src/pages/accounts/NewPasswordForm.js [hunk 1]
```diff
@@ -23,97 +28,140 @@ function ResetNewPassword() {
...
+                  {loading ? 'SENDING...' : 'SUBMIT'}
+                </button>
+              </form>
+            </>
+          ) : (
+            <div>
+              {' '}
+              <>
+                <h3 className="text-2xl font-bold text-center py-3 font-rale">
+                  Invalid Token
```
Comment (samyshyaka) [inline src/pages/accounts/NewPasswordForm.js]: I think we can change this message to something else that the user can understand. The user may not understand what the token is.

**Review by samyshyaka — CHANGES_REQUESTED** [review]
This is great work @Yvad60, here below is some feedback.

**Review by feyton — DISMISSED** [review]
### Approval
This is really great work @Abdulkeza and @Yvad60 
Try to fix the issue with codeclimate but since the feature is well integrated.

**Review by samyshyaka — APPROVED** [review]

**Review by feyton — APPROVED** [review]
### Approval
This really great work @team

**Pull request merged**